crates/file_finder/src/file_finder.rs

```diff
@@ -424,7 +424,8 @@ impl FileFinderDelegate {
             match event {
                 project::Event::WorktreeUpdatedEntries(_, _)
                 | project::Event::WorktreeAdded
-                | project::Event::WorktreeRemoved(_) => file_finder
+                | project::Event::WorktreeRemoved(_)
```

Comment:
File finder's contents does not change after sorting the wortkrees, so we don't have to reload it on sort?

Reply:
I wasn't sure on this one, happy to remove it and test again but I wanted to confirm which part of Zed that is for certain. It's the pop-up search on `Cmd+P` right?

Reply:
No problem, I will drop this one back out :) Thanks!